core/src/test/java/org/apache/spark/util/collection/TestTimSort.java

```diff
@@ -0,0 +1,134 @@
+package org.apache.spark.util.collection;
+
+import java.util.*;
+
+/*
```

Comment:
you need to put this in the beginning of the file, i.e. before the package definition